mock_service/doc/pydoc/action.md

```diff
@@ -0,0 +1,60 @@
+<a id="lib.action"></a>
```

Comment:
Do we possibly need a small README.md in this directory that can explain how the files can be regenerated, and that they should be updated if you change something in corresponding *.py files.

Reply:
I added a generation section in the README